src/main/java/seedu/foodrem/views/StringView.java

```diff
@@ -0,0 +1,24 @@
+package seedu.foodrem.views;
+
+import javafx.scene.Node;
+import javafx.scene.control.TextArea;
+
+/**
+ * A view of a {@code String}. This can be displayed.
+ * @author Richard Dominick
+ */
+public final class StringView {
+    private StringView() {} // Prevents instantiation
+
+    /**
+     * Creates a new view of the given string.
+     * @param message the string to be displayed.
+     * @return the node to be displayed in the UI.
+     */
+    public static Node from(String message) {
+        final TextArea textArea = new TextArea(message);
+        textArea.setEditable(false);
+        textArea.setWrapText(true);
```

Comment:
I believe we can use a HBox for this as well. HBox offers wrapping text as well. This will then allow the style of the string to match those of which in other command results

Reply:
Will resolve this in a future PR

Reply:
The reason I chose `TextArea` was because it means the output text would be selectable by the user should they e.g. want to copy and paste for their own reference/convenience. Actually @Eugenetanwl3881 do you think we should include this point in the UG?

Reply:
If we are using this, I think we can include this as a note (perhaps in the Layout Section) of the UG? If mentioning it is not that important, leaving it out of the UG shouldn't be an issue as well.

Reply:
Yup sure, we can put a `` ```note `` admonition :)